test/integration/backward_compatible/parallel/serialization/DefaultSerializersLiveTest.js

```diff
@@ -34,7 +34,10 @@ describe('DefaultSerializersLiveTest', function () {
         client = await testFactory.newHazelcastClientForParallelTests({
             clusterName: cluster.id
         }, member);
-        map = await client.getMap('test');
+    });
```

Comment:
Not related to this test, but can you also update the binary compatibility tests by generating the binary file again with those newly added serializers?

Reply:
only java class serializer is added and I cannot find `Object[]` object in it.

Reply:
You can add it by yourself I guess

Reply:
added in https://github.com/hazelcast/hazelcast-nodejs-client/pull/1197/commits/ac047bc15708dc2a3ea2bcf8066801dfbcff31b4